src/components/actorScriptButton.jsx

```diff
@@ -11,9 +11,9 @@ const ActorScriptButton = ({
 }) => {
   // on change, change the value of current actor
   const onClickChange = (e) => {
-    const value = e.target.value.split(' ');
-    const firstName = value[0];
-    const lastName = value[1];
+    const value = e.target.value.split(' '); // todo, use actorId here instead of string manipulation
+    const firstName = value.at(0);
+    const lastName = value.at(-1);
```

Comment:
When the name `"Jeff "`   `"peden"` was used, the previous code would split on spaces and take the first two elements. 
```js
value[0] = "Jeff"
value[1] = " "
value[2] = "peden"
```
This resulted in us mistakenly navigating to the `/currentCharacters/jeff//test` endpoint instead of the correct `/currentCharacters/jeff/peden/test`.  

I've put in a hack to use the first and last values for names until we migrate to using actorIds

Reply:
Good catch, thank you! I'm in the process of transitioning to actorIds right now.